utils/server-side-helper/app/get-url.ts

```diff
@@ -0,0 +1,7 @@
+export const getURL = (path: string = '') => {
+  const baseURL = process.env.NEXT_PUBLIC_BASE_URL || 'http://localhost:3000';
```

Comment:
Idem for default url. You could even declare a constant `prodBaseUrl` here https://github.com/annuaire-entreprises-data-gouv-fr/site/blob/08f8511c0e90b82ed4e191bb81419483d7ca423f/models/constants.ts#L1

Reply:
Actually not sure`prodBaseUrl` is a good idea